proxysql/assets/dashboards/-_overview.json

```
@@ -0,0 +1,377 @@
+            "x": 75,
+            "y": 10
+        },
+        {
+            "height": 13,
+            "id": 9,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:proxysql.query_cache.memory_bytes{$proxysql_server,$proxysql_port}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Memory usage of the query cache",
+            "type": "timeseries",
+            "width": 47,
+            "x": 75,
+            "y": 26
+        },
+        {
+            "height": 13,
+            "id": 10,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:proxysql.query_cache.bytes_out{$proxysql_server,$proxysql_port}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Cache read throughput",
+            "type": "timeseries",
+            "width": 47,
+            "x": 75,
+            "y": 42
+        },
+        {
+            "columns": "[\"host\", \"service\", \"source\"]",
+            "height": 55,
+            "id": 12,
+            "indexes": [],
+            "message_display": "expanded-md",
+            "query": "source:proxysql",
+            "show_date_column": true,
+            "show_message_column": true,
+            "sort": {
+                "column": "time",
+                "order": "desc"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Log Stream",
```

(nit) to match other title case

```suggestion
            "title_text": "Log stream",
```